dipy/segment/clustering.py

```diff
@@ -416,7 +417,7 @@ class QuickBundles(Clustering):
     metric : str or `Metric` object (optional)
         The distance metric to use when comparing two streamlines. By default,
         the Minimum average Direct-Flip (MDF) distance [Garyfallidis12]_ is
-        used and requires streamlines to have the same number of points.
+        used and streamlines are automatically resampled so they have 18 points.
```

Comment:
I assume there's some reason 18 is used here?

Reply:
I want to make a note here for future reference: I found a reason to increase the resolution. In PR #1625 I'm using quickbundles to generate templates based on a set of many bundles. The centroids in aggregate form the bundle template, and the resolution of the centroid is this nb_points so if you want a higher resolution/nicer template there is reason to change this. I wasn't looking at the example initially and found this confusing (maybe it's just me? I've been using quickbundles for awhile so I wasn't as quick to reference the docstring example as I should have been). Does it make sense to add a few words in the input description of metric to make sure the example gets referenced when someone is using quickbundles?